test/acceptance/specs/RevokeApplicationAuthoritySpec.scala

```diff
@@ -52,7 +53,7 @@ class RevokeApplicationAuthoritySpec extends BaseSpec with NavigationSugar {
       clickOnSubmit()
 
       on(PermissionWithdrawnPage(app.application.id))
-      verifyText(withdrawnMessageText, s"${app.application.name} no longer has authority to interact with HMRC on your behalf.")
+      verifyText(withdrawnMessageText, "This application no longer has authority to interact with HMRC on your behalf.")
```

Comment:
Was this the solution that Alex/UX settled on?  

Have we explored the alternative of having the POST reply containing the success message? Especially since this form submission is (likely) idempotent, it may make sense.